### ISSUE TITLE
null in top10_contributors

### ISSUE THREAD

**Issue by kitplummer** [gtri/lowendinsight#46]
https://github.com/pouchdb/pouchdb

```
{
        "header": {
          "uuid": "33e0def0-77a4-11ea-b7d2-82dd17abe001",
          "start_time": "2020-04-06T01:16:04.080788Z",
          "source_client": "mix task",
          "repo": "https://github.com/pouchdb/pouchdb",
          "library_version": "",
          "end_time": "2020-04-06T01:16:11.668128Z",
          "duration": 7
        },
        "data": {
          "risk": "low",
          "results": {
            "top10_contributors": [
              {
                "name": "Dale Harvey",
                "contributions": 930
              },
              {
                "name": "Nolan Lawson",
                "contributions": 927
              },
              {
                "name": "Dale",
                "contributions": 317
              },
              null,
              {
```

**Comment by kitplummer** [comment]
Need to filter all "bad" entries in the git log before processing.

**Comment by kitplummer** [comment]
Looks like this:

```
["930", "Dale Harvey"]
["927", "Nolan Lawson"]
["317", "Dale"]
["Harvey"]
["263", "greenkeeper[bot]"]
["250", "Calvin Metcalf"]
```

https://github.com/gtri/lowendinsight/blob/develop/lib/git_module.ex#L217

**Comment by kitplummer** [comment]
Looks to be the regex for handling the `git shortlog`.

Testing: `String.split(~r{[0-9]+\s\s})` now.

**Comment by kitplummer** [comment]
resolved with #47
